SafeView.web.js

```diff
@@ -0,0 +1,437 @@
+import * as React from 'react';
+import {
+  View,
+  InteractionManager,
+  Dimensions,
+  Animated,
+  StyleSheet,
+} from 'react-native';
+import hoistStatics from 'hoist-non-react-statics';
+import { doubleFromPercentString } from './utils';
+// https://github.com/zhetengbiji/safeAreaInsets/blob/master/src/index.ts
+const attrs = ['top', 'left', 'right', 'bottom'];
+
+function getSupportedTransitionEvent() {
+  const el = document.createElement('invalidtype');
+  const transitions = {
+    transition: 'transitionend',
+    OTransition: 'oTransitionEnd',
+    MozTransition: 'transitionend',
+    WebkitTransition: 'webkitTransitionEnd',
+  };
+
+  for (const t in transitions) {
+    if (el.style[t] !== undefined) {
+      return transitions[t];
+    }
+  }
+}
+
+function getSupportedEnv() {
```

Comment:
This check could probably be as simple as `env` by default and `constant` is supported